node-fs-readfile/cat.js

```diff
@@ -0,0 +1,7 @@
+const fs = require('fs');
+for (let i = 2; i < process.argv.length; i++) {
+  fs.readFile(`./${process.argv[i]}`, 'utf8', (err, data) => {
+    if (err) throw err;
+    console.log(data);
+  });
+}
```

Comment:
Try this again with the really huge file first, multiple times. Like 5 times in a row. It's possible that the large file prints last.